tools/benchmark_tool/openvino/tools/benchmark/main.py

```diff
@@ -278,6 +278,10 @@ def set_throughput_streams():
             def set_nthreads_pin(property_name, property_value):
                 if property_name in supported_properties or device_name == AUTO_DEVICE_NAME:
                     # create nthreads/pin primary property for HW device or AUTO if -d is AUTO directly.
+                    if property_value == "YES":
```

Comment:
Please move this to the beginning of set_nthreads_pin and if property_name == 'AFFINITY':